Automattic/jetpack
# Site Verification Tools: Add Facebook Domain Verification

Fixes #19548

#### Changes proposed in this Pull Request:

* Adds Facebook Domain Manager to the list of services covered by the Site Verification Tools

#### Jetpack product discussion


All discussion is in the original issue which has inspired this PR

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Visit `/wp-admin/admin.php?page=jetpack#/traffic`
* Verify that "Facebook" appears under Site Verification Tools
* Enter a string (`rvv8b23jxlp1lq41I9rwsvpzncy1fd` is one which I found online slightly adjusted)
* Save, and check the site. Verify that the service appears in the mark-up. 

<img width="594" alt="Screenshot 2021-05-01 at 19 15 53" src="https://user-images.githubusercontent.com/43215253/116793859-389aa100-aac1-11eb-834b-2029f23ce3b7.png">
